templates/2024/navbar.html

```diff
@@ -38,9 +38,10 @@
                 <li class="dropdown-item">
                   <a class="nav-link g-px-0 g-color-black" href="/2024/about/team/">Team</a>
                 </li> 
-                <li class="dropdown-item">
+                <!-- Commented out some links in the About dropdown as they are catered for in the Attendee Guide dropdown now -->
```

Comment:
We could as well delete this(the comment plus the list items) as is dead code since is catered for under `Attendee Guide`